src/components/ChatPopUp.tsx

```diff
@@ -93,6 +93,10 @@ export interface ChatPopUpProps
    * Whether to show the initial message popup when the panel is hidden on load.
    * Defaults to false.
    */
+
+  /** The bot id of the current bot. Defaults to the placeholder "BOT_ID_HERE" */
+  botId?: string;
```

Comment:
I think we don't necessarily need to correlate this open on load (or not) behavior with specific bot. As long as user click the close button on ChatPopUp, we will just note in local storage that it should remain close on load.

(if there is a situation we want to be bot specific, then we should get the botId from state/chat-headless instead of asking user to provide the botId in prop)

Reply:
Removed this prop